nautobot/core/filters.py

```diff
@@ -15,10 +15,9 @@
 from drf_spectacular.utils import extend_schema_field
 from taggit.managers import TaggableManager
 
-from nautobot.dcim import fields as dcim_fields
-from nautobot.dcim import forms as dcim_forms
-from nautobot.extras import models as extras_models
-from nautobot.utilities import constants, forms, utils
+from nautobot.core import constants, forms
+from nautobot.core.models import fields as core_fields
+from nautobot.core.utils import data as data_utils
```

Comment:
Why did the `dcim` app disappear here? Was utilities importing core forms via DCIM?

Reply:
Nice spot! Yeah, we were pulling in `MACAddressField` (form field) and `MACAddressCharField` (model field) from DCIM. I ran into some dependency issues around that which I fixed by moving those two classes from DCIM into core.